pac4j-saml/src/test/java/org/pac4j/saml/client/RedirectSAML2ClientTests.java

```diff
@@ -12,18 +10,10 @@
 import org.springframework.mock.web.MockHttpServletRequest;
 import org.springframework.mock.web.MockHttpServletResponse;
 
-import java.io.BufferedReader;
-import java.io.ByteArrayInputStream;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.nio.charset.StandardCharsets;
 import java.util.Arrays;
-import java.util.Base64;
-import java.util.List;
-import java.util.zip.Inflater;
-import java.util.zip.InflaterInputStream;
 
 import static org.junit.Assert.*;
+import static org.pac4j.saml.client.AuthnRequestInflator.getInflatedAuthnRequest;
```

Comment:
Remove, and instead qualify with `AuthnRequestInflator.getInflatedAuthnRequest()` where needed.

Reply:
OK